Mathlib/CategoryTheory/MorphismProperty/Tactic.lean

```diff
@@ -0,0 +1,231 @@
+/-
+Copyright (c) 2024 Andrew Yang, Calle Sönne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang, Calle Sönne
+-/
+import Mathlib.CategoryTheory.MorphismProperty.Basic
+
+/-!
+# The `addMorphismPropertyInstances` command
+
+We define a command `addMorphismPropertyInstances` that goes through all the lemmas tagged with
+`morphismPropertyInstance` and attempts to add the relevant instances of a morphism property,
+provided that the appropriate stability instances on the morphism property are present.
+
+-/
+open CategoryTheory
+
+open Lean Meta Elab Tactic
+open Mathlib.Tactic
+
+namespace AddMorphismPropertyInstances
+
+/-- Configuration options for `@[morphisProperty]` -/
+structure Config where
+  /-- The priority of the generated lemma -/
+  priority : Nat := 1000
+  deriving Inhabited
+
+/-- Function elaborating `Config` -/
+declare_command_config_elab elabConfig Config
+
+/-- The syntax for the `morphismPropertyInstance` attribute. -/
+syntax (name := morphismPropertyInstance) "morphismPropertyInstance" Parser.Tactic.optConfig : attr
```

Comment:
Do we have an established convention for naming such things, mk_iff_of_inductive_prop is the closest command I can think of and is different